pkg/controller/config/config_controller.go

```diff
@@ -256,10 +265,23 @@ func (r *ReconcileConfig) replayData(ctx context.Context, w *watch.Set) error {
 		}
 
 		for i := range u.Items {
+			syncKey := strings.Join([]string{u.Items[i].GetNamespace(), u.Items[i].GetName()}, "/")
```

Comment:
We probably want to use a shared function for setting syncKey, to avoid code duplication.

Reply:
added `GetSyncKey`